charts/prometheus-smartctl-exporter/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.0
+version: 0.6.1
```

Thank you, @nepomucen, for your PR. Please, bump a minor number with a new feature.

```suggestion
version: 0.7.0
```